packages/client/src/rpc/modules/engine/util/getPayloadBody.ts

```diff
@@ -0,0 +1,14 @@
+import { bytesToHex } from '@ethereumjs/util'
```

Comment:
Any objection to including under `getPayload.ts` since it and the `getPayload` methods are logically connected and this doesn't have many lines of code to it.